src/tools/roc_recv/main.cpp

```diff
@@ -263,39 +265,54 @@ int main(int argc, char** argv) {
         return 1;
     }
 
-    if (args.source_given == 0 && args.repair_given == 0) {
-        roc_log(LogError, "at least one --source or --repair port should be specified");
+    if (!args.source_given && !args.repair_given) {
+        roc_log(LogError, "at least --source or --repair port should be specified");
```

Comment:
Actually it can be `if (!args.source_given) {`. We can't work without a source port.

Reply:
Done